src/components/ShowList.css

```diff
@@ -26,3 +28,14 @@
   border: solid 2px black;
   border-radius: 2px;
 }
+
+.delete-button {
+  text-align: right;
+}
+
+.success-message {
+  display: block;
+  margin: 10px auto;
+  color: green;
+  font-weight: 600;
+}
```

Comment:
I've just reviewed the other pairs code and they've now created an Item.css file - perhaps we should consider moving all Item specific classes into an Item.css file now, so that our code aligns when merged ☺️

Reply:
I'd recommend waiting until one of the PRs are merged first, then merge main into the remaining one and resolve any conflicts there. 👍